src/test/java/com/mongodb/jdbc/MongoSQLResultSetMetaDataTest.java

```diff
@@ -0,0 +1,260 @@
+package com.mongodb.jdbc;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.SQLException;
+import java.sql.Types;
+import java.sql.ResultSetMetaData;
+import org.bson.BsonValue;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class MongoSQLResultSetMetaDataTest extends MongoSQLMock {
+    private static MongoResultSetMetaData resultSetMetaData;
+
```

Comment:
note while it says `__bot` here to make the comment easier to read, below in the `getTable` test we actually get `""` as the result.